doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -257,6 +257,30 @@ ifdef::PAM[]
 
 == Process designer
 
+.Validation for a signal name fails if the name includes spaces or colons (`:`) [https://issues.redhat.com/browse/RHPAM-2557[RHPAM-2557]]
+
+Issue: In the process designer, the validation for a signal name fails if the name includes spaces, colons (`:`), or other special characters.
+
+Steps to reproduce:
+
+. Open a BPMN project in the process designer.
+. Add a `Signal` node on the canvas.
+. In the *Properties* panel for the node, expand *Implementation/Execution*.
```

Comment:
Well, there is only one Properties panel. However the properties change depending on what has focus. I would do this:
Select the node and then in the Properties panel, expand *Implementation/Execution*.

Reply:
This is intentional equivocation. When I run it, the panel is open immediately when I create the node, but it might not always be the case. In one case I tried when the Properties panel was closed and I did not even know how to reopen it - selecting the node did not work. To avoid having to cover all cases of opening the panel or selecting the node or doing nothing as it's already open for the node, I use a formula that covers all cases and assume they will know how to open the node's properties